R/mod_settingsCode.R

```diff
@@ -126,29 +145,24 @@ settingsCode <- function(input, output, session, mapping, charts){
 
   # initApp.R
   initString<- paste(
+    "#####################################################################################################################",
+    "# Initialize the safetyGraphics Shiny App using exported settings and data",
+    "#",
+    "# This code was generated in the safetyGraphics Shiny App to help re-initialize the app with the current settings.",
+    "# The code assumes that `mapping.yaml`, `charts.yaml` and `domainData.RDS` were downloaded from the app and saved ",
+    "# in you working directory. It also assumes that the default meta data included in the app is used. However, there ",
+    "# are likely many use cases where this workflow is not ideal and some customization is needed.  See ?safetyGraphicsApp ",
+    "# and the pacakge vignettes for more details and customization options.",
+    "#",
+    "#####################################################################################################################",
+    "",
     "library(safetyGraphics)",
+    "library(safetyCharts)",
```

Comment:
How do you see this extending for other packages that are creating extensions for safetyGraphics (like Tplyr)? Could we have some logic to determine what packages the configs are coming from and loading those in?

Reply:
Yeah - we should probably be scraping the package field in the charts object and adding libraries dynamically. Something like: 

```
libs <- charts %>% map(~.x$package) $>$ unique
libText<- paste("library(",libs,")")
# Paste libText in to initString
```
 
I'll start an issue to track this update.